### ISSUE TITLE
Gather resources for contribution intro

### ISSUE THREAD

**Issue by angelamnr** [drupaldiversity/contrib#2]
Need to start gathering resources for a beginner's guide to contributing to Drupal. A good starting point is, of course, the [official page](https://www.drupal.org/getting-involved-guide). If we get something easily digestible started here, we could contribute back to d.o or other groups.

Some thoughts:
- I'd like to ask the people at the Drupal Dojo and the mentoring channels how they onboard new mentees to contributing
- Probably easiest to start with a list of requirements needed to start contributing since a decent portion of the process is setting things up in your d.o account
- We should be mindful of the posting limitations that new d.o accounts have. I remember being unsure when my first issue could be viewed by other users, which is tough when you're an impatient newbie. 😛

**Comment by sparklingrobots** [comment]
I haven't looked at it in a long time but there's also drupalladder.com. 

If it would be helpful to have the new folks share what they want to learn, we can tag them in channel to come add their thoughts--don't want to swarm your thread with questions if they aren't useful, though. :)

**Comment by zenlan** [comment]
This came through my inbox today and it contains a few nuggets of distilled wisdom imo.

As a template  for building/organising the resources in the ddi contrib repos I think that the list below is a worthy suggestion. Lack of tech jargon is a +1.

[Successful Volunteer-based Software Projects](https://medium.com/@adam.kariv/successful-volunteer-based-software-projects-fad6d294ad07)

**tl;dr**
>1. As a new contributor, I open the project’s home page for the first time. 
  How long does it take me to understand what the project does?
> 2. I decide I want to contribute. 
  Can I quickly find an open task that’s suited for my skills and expertise? 
> 3. I found a proper task.
  How fast can I understand what exactly needs to be done here?
> 4. I know what needs to be done.
  How easy it is to get the code, find the location that needs to be changed and fix it?
> 5. I made a modification to the code.
  How much time do I spend in creating a development environment to try it out? 
  How simple is it to add verification tests?
> 6. I verified my change and want to submit it.
  How long does it take me to figure out how to do it?
  How long do I wait for that change to be approved?
> 7. My change was approved. 
  When will I see it used / released / deployed?
>

**Comment by angelamnr** [comment]
@sparklingrobots Drupal Ladder is a great resource, thanks! I think this page in particular will be really useful: http://drupalladder.org/learn-sprints
Yes, reaching out to new folks would be great! I don't want to accidentally miss something that someone wanted to learn about.

@zenlan That's a super useful article. Those steps are so useful! Definitely a good reminder to talk about the beginning and the end of the contribution process, especially since it can be hard to find the issue queue if you don't know where to look. Thanks for sharing.

**Comment by laughnan** [comment]
Going to close this. We're moving all issues onto Drupal.org: https://www.drupal.org/project/issues/ddi_contrib
